ViewPacker/ViewPacker.csproj

```diff
@@ -64,8 +64,12 @@
     <PreBuildEvent>npm install --no-bin-links --prefix "$(ProjectDir)tools" "$(ProjectDir)tools"</PreBuildEvent>
   </PropertyGroup>
   <PropertyGroup>
-    <PostBuildEvent>"$(ProjectDir)..\packages\Node.js.redist.10.16.3\tools\win-x64\node.exe" "$(ProjectDir)tools\node_modules\node-sass\scripts\install.js"
+    <PostBuildEvent Condition="'$(OS)' == 'Windows_NT'">"$(ProjectDir)..\packages\Node.js.redist.10.16.3\tools\win-x64\node.exe" "$(ProjectDir)tools\node_modules\node-sass\scripts\install.js"
```

Comment:
you may have only one and replace the node with the ambient node in this case

Reply:
After speaking to master @alvesmiguel1 , we realised that the best option is to have static vendor files since there's apparently no way of generating the appropriate darwin binding files from within Windows. Check my latest commit please.